test/TestGrains/StatelessWorkerGrain.cs

```diff
@@ -60,14 +59,12 @@ private static Task TimerCallback(object state)
         }
 
 
-        public Task<Tuple<Guid, List<Tuple<DateTime, DateTime>>>> GetCallStats()
+        public Task<Tuple<Guid, string, List<Tuple<DateTime, DateTime>>>> GetCallStats()
         {
             Thread.Sleep(200);
-            lock (allActivationIds)
```

Comment:
Original logging locked allActivationIds, new code does not.  Sure this is safe?